src/main/java/com/ft/methodearticlemapper/resources/PostContentToTransformResource.java

```diff
@@ -88,7 +90,9 @@ private void validateUuid(String uuid, EomFile eomFile) {
             throw ClientError.status(400).context(null).reason(ErrorMessage.UUID_REQUIRED).exception();
         }
         try {
-
+            if (!uuidPattern.matcher(uuid).matches()) {
+                throw new IllegalArgumentException();
```

Comment:
A general message is added in the rethrow: "The UUID passed was invalid", I could add some message here also, but what else can I say than that? It will be duplicate...